src/main/java/com/google/sps/servlets/SearchServlet.java

```diff
@@ -41,27 +42,44 @@
   @Override
   /* Show courses. */
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    List<Course> courses = getHelper(request);
-
-    String coursesJson = new Gson().toJson(courses);
+    JSONObject results = getMatchingCourses(request);
```

Comment:
I changed it to results because it holds courses and a message, but I think you're right in that courseResults is still a better name than results in this case. Thanks!